schemas/log.yml

```diff
@@ -2,8 +2,14 @@
 - name: log
   title: Log
   group: 2
+  short: Low level details specific to log events.
   description: >
-    Fields which are specific to log events.
+    Low level details specific to log events.
+
+    `log.*` typically contains lower level details about the logging mechanism
```

Comment:
Suggest: "Details about the event's logging mechanism.  The `log.*` fields are typically populated with details about the logging mechanism used to create and/or transport the event.  For example, syslog details are typically mapped here.  Note that details about the event that is transported over syslog are not typically stored here, but are instead stored in the `event.*` or other ECS fields.

Optional:  <why not invest some text in making this super clear, by adding a more complete and correct sentence, like:>   "When mapping syslog details, the following mappings are recommended:  facility->`log.facility`, severity name->`log.level`, `severity number`->`log.severity`, <add them all>)"

Reply:
I'm putting in place the first suggestion. With the latest developments in #525, I think we won't need the optional idea. :-)